graylog2-web-interface/src/components/configurations/PermissionsConfig.tsx

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import * as React from 'react';
+import { useState } from 'react';
+import styled, { css, DefaultTheme } from 'styled-components';
+import { Form, Formik } from 'formik';
+import { PermissionsConfigType } from 'src/stores/configurations/ConfigurationsStore';
+
+import { Button, Col, Modal, Row } from 'components/bootstrap';
+import FormikInput from 'components/common/FormikInput';
+import Spinner from 'components/common/Spinner';
+import { InputDescription } from 'components/common';
+
+type Props = {
+  config: PermissionsConfigType,
+  updateConfig: (config: PermissionsConfigType) => Promise<void>,
+};
+
+const StyledDefList = styled.dl.attrs({
+  className: 'deflist',
+})(({ theme }: { theme: DefaultTheme }) => css`
+  &&.deflist {
+    dd {
+      padding-left: ${theme.spacings.md};
+      margin-left: 200px;
+    }
+  }
+`);
+
+const LabelSpan = styled.span(({ theme }: { theme: DefaultTheme }) => css`
+  margin-left: ${theme.spacings.sm};
+  font-weight: bold;
+`);
+
+const PermissionsConfig = ({ config, updateConfig }: Props) => {
+  const [showModal, setShowModal] = useState<boolean>(false);
+
+  const _saveConfig = (values) => {
+    updateConfig(values).then(() => {
+      setShowModal(false);
+    });
+  };
+
+  const _resetConfig = () => {
+    setShowModal(false);
+  };
+
+  return (
+    <div>
+      <h2>Permissions Config</h2>
+      <p>These settings can be used to control which entity sharing options are available</p>
```

Comment:
We are missing a dot here:
```suggestion
      <p>These settings can be used to control which entity sharing options are available.</p>
```